.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+<!-- Be sure you are following the instructions in the dev guidelines
+http://www.numpy.org/devdocs/dev/gitwash/development_workflow.html -->
+
+<!-- Include a properly formatted commit message
```

Comment:
Can you change "Include a properly" to "We'd appreciate it if you would include a properly ..."